File: node/plans.js
const axios = require("axios")

const hostname = "api.sanctions.io"
const bearerToken = "12345678-12AF-12aF-12aF-1234567890aF"
const apiVersion = "2.1"

let uri = `https://${hostname}/plans`
axios
  .get(uri, {
    headers: {
      Authorization: `Bearer ${bearerToken}`,
      Accept: `application/json; version=${apiVersion}`,
    },
  })
  .then((response) => {
    let json = response.data
    let results = json.results
    let planNames = results.map((plan) => plan["plan_name"])
    console.log(`Plans found: ${planNames.join(", ")}.`)
  })

File: python/plans.py
import logging

import requests

HOSTNAME = "api.sanctions.io"
BEARER_TOKEN = "12345678-12AF-12aF-12aF-1234567890aF"
API_VERSION = "2.1"

logging.basicConfig()
logger = logging.getLogger()
logger.setLevel(logging.INFO)


def invoke_plans():
    """
    Example showing how to call the /plans endpoint.
    :return: dict
    """
    url = f"https://{HOSTNAME}/plans"
    headers = {
        "Authorization": f"Bearer {BEARER_TOKEN}",
        "Accept": f"application/json; version={API_VERSION}",
    }

    response = requests.get(url, headers=headers)
    return response.json()


if __name__ == "__main__":
    json_data = invoke_plans()
    results = json_data.get("results", [])
    names = [result.get("plan_name") for result in results]
    logger.info(f"Plans found: {','.join(names)}")
